oauthenticator/generic.py

```diff
@@ -154,15 +154,24 @@ async def update_auth_model(self, auth_model):
         the user isn't part of `admin_users` or `admin_groups`. Note that
         leaving it at None makes users able to retain an admin status while
         setting it to False makes it be revoked.
+
+        Also populates groups if `manage_groups` is set.
         """
+        # Authenticator.manage_groups is new in jupyterhub 2.2
```

Comment:
@yuvipanda @minrk Do you think we should continue supporting JupyterHub 1, or could we bump the minimum version?

Reply:
Perfectly fine to bump I think